src/components/Scale.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect } from 'react'
+import { useMapInstance } from '@datapunt/react-maps'
+import { Control, control } from 'leaflet'
+
+import ScaleOptions = Control.ScaleOptions
```

Comment:
This is a syntax I am not familiar with, what is it?

Reply:
My IDE did this, apparently you can import types like this from a namespace. Removed it for now and directly added the type to the prop